tools/pyorac/arguments.py

```diff
@@ -241,17 +242,17 @@ def args_cc4cl(parser):
     phs = cccl.add_mutually_exclusive_group()
     phs.add_argument('-s', '--settings', type=str, action='append',
                      default=None, help='Parameters for each phase to be '
-                     'processed. Each element is a string listing all the '
-                     'arguments to be applied for that phase. This is processed '
-                     'using the same parser so all arguments listed for the '
-                     'main processor are available.')
+                                        'processed. Each element is a string listing all the '
```

Comment:
Ugh. This spacing is *so inefficient* on a short line length but this will look like trash on my editor.
I shall have to pull a copy and redo all the line breaks. <grumble>

Reply:
Updated!